admin_guide/high_availability.adoc

```diff
@@ -356,7 +357,7 @@ $ oc set env dc/ipf-ha-router \
 $ oc volume dc/ipf-ha-router --add --overwrite \
     --name=config-volume \
     --mount-path=/etc/keepalive \
-    --source='{"configMap": { "name": "mycustomcheck"}}'
+    --source='{"configMap": { "name": "mycustomcheck", "defaultMode": 493}}'
```

Comment:
"defaultMode": 0755 does not work.
error: invalid character '7' after object key:value pair